### DISCUSSION TITLE
image prompt won't work in a colab

### DISCUSSION THREAD

**Discussion by lsooxlla8** [lllyasviel/Fooocus#674] [Q&A]
Hi there!
I am trying to use image prompt function on a free tier colab, but every time i hit generate it gives me an error and colab cell just stops itself with the follwing:
<img width="1511" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/41743965/d6176eaf-de63-4c4b-856c-b1b894654657">

**Comment by barepixels** [comment]
copy paste

Free colab limited memory and image prompt feature requires more? I faced a similar problem when tried using free colab to make loras (during generation it would get a connection timeout error). Trying to sell my kidney, hopefully, I will get a used 3090 by next week.
